kademlia/protocol.py

```diff
@@ -108,9 +116,9 @@ def handleCallResponse(self, result, node):
         """
         if result[0]:
             self.log.info("got response from %s, adding to router" % node)
-            self.router.addContact(node)
             if self.router.isNewNode(node):
```

Comment:
I think this line and the next could just be replaced with a call to ```maybeTransferKeyValues``` for consistency.  Also, if the pattern is always call ```maybeTransferKeyValues``` which is only two lines which then may call ```transferKeyValues``` - then we could probably just consolidate those two methods into a single method.  My name is crappy - maybe something more like ```sendRelevantKeysIfNew``` or something like that.  Thoughts?